platform/net/ubuntu_net_manager.go

```diff
@@ -89,20 +89,24 @@ func (net UbuntuNetManager) SetupIPv6(config boshsettings.IPv6, stopCh <-chan st
 
 func (net UbuntuNetManager) SetupNetworking(networks boshsettings.Networks, mbus string, errCh chan error) error {
 	if networks.IsPreconfigured() {
-		err := net.setupFirewall(mbus)
+
+		// Note in this case IPs are not broadcast
+		dnsNetwork, _ := networks.DefaultNetworkFor("dns")
+		net.logger.Info(UbuntuNetManagerLogTag, "Successfully set up outgoing nats api firewall")
+		err := net.writeResolvConf(dnsNetwork.DNS)
+		if err != nil {
+			return err
```

Comment:
Is the DNS configuration order changed because of the `hostname` use case for NATs?

Reply:
I have just put skeleton code where I assumed I'd have to call the iptables logic when I created the windows pr. I noticed that dns probably should come first and moved it when I actually implemented it.